src/PresentationalComponents/Common/Tables.js

```diff
@@ -1,6 +1,8 @@
 import { generateFilter } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import { SYSTEM_TYPES } from '../../AppConstants';
 
+// All this feels like it would be better in a helpers file or something.
```

Comment:
Most of the functions are all helpers used inside different tables (Pathways,Rules,Systems tables) and inventory component. Others Like workloadQueryBuilder are used in 11 files. In functions that are used across more than just table components, what if we put those in a 'AppHelpers file' and then keep the functions relative to just tables here, but rename this file 'TableHelpers